packages/fbjs/src/core/hyphenateStyleName.js

```diff
@@ -31,7 +33,10 @@ const msPattern = /^ms-/;
  * @return {string}
  */
 function hyphenateStyleName(string) {
-  return hyphenate(string).replace(msPattern, '-ms-');
+  if (memoized[string] === undefined) {
+    memoized[string] = hyphenate(string).replace(msPattern, '-ms-');
+  }
+  return memoized[string];
```

Comment:
Practically this seems fine. But let's use [`memoizeStringOnly`](https://github.com/facebook/fbjs/blob/4369c7dfeb1ab0de27feb3e9245f599c8edd6a5c/packages/fbjs/src/core/memoizeStringOnly.js) (`module.exports = memoizeStringOnly(hyphenateStyleName)`). That'll also make sure we don't do the wrong thing for `'hasOwnProperty'` 😜

Reply:
ok!